sites/fast-website/webpack.prod.js

```diff
@@ -12,12 +12,12 @@ module.exports = merge(baseConfig, {
         minimizer: [new TerserJSPlugin({}), new OptimizeCSSAssetsPlugin({})],
     },
     output: {
-        filename: "[name].[hash].js",
+        filename: "bundle/[name].[hash].js",
     },
     plugins: [
         new MiniCssExtractPlugin({
-            filename: "[name].[hash].css",
-            chunkFilename: "[id].[hash].css",
+            filename: "bundle/[name].[hash].css",
+            chunkFilename: "bundle/[id].[hash].css",
```

Comment:
Could I see a picture of the output files so I can make sure Azure Front Door and CDN are configured appropriately?

Reply:
The full paths would be:
```
sites/fast-website/dist/index.html
sites/fast-website/dist/bundle/main.[hash].css
sites/fast-website/dist/bundle/main.[hash].js
sites/fast-website/dist/bundle/main.[hash].js.LICENSE.txt
```

These files are then copied to:
```
sites/website/static/index.html
sites/website/static/bundle/main.bf2fec3c322fba38ac59.css
sites/website/static/bundle/main.bf2fec3c322fba38ac59.js
sites/website/static/bundle/main.bf2fec3c322fba38ac59.js.LICENSE.txt
```

Reply:
That works great.  Those files will get cached at the Azure Front Door.